components/folders/folderRoutes.js

```diff
@@ -1,12 +1,25 @@
 const router = require('express').Router()
 const controllers = require('./folderControllers')
+const utils = require('../../utils')
 
-router.post('/', controllers.createFolder)
+router.post('/', utils.isRequestAuthenticated, controllers.createFolder)
```

Comment:
`router.post('/', utils.isRequestAuthenticated, controllers.createFolder)`
@czclaxton @luiscmartinez 

Okay for now, but need to move away from authenticating on each route. This should be a middleware that is called once before entering any protected route. We e only want to protect routes that write/update database or protect sensitive information.